src/Mailjet/Request.php

```diff
@@ -56,26 +58,27 @@ public function __construct($auth, $method, $url, $filters, $body, $type)
      * @param $call
      * @return Response the call response
      */
-    public function call($call)
-    {
+    public function call($call) {
         $payload = [
-            'headers'  => ['content-type' => $this->type],
+            'headers' => ['content-type' => $this->type],
             'query' => $this->filters,
             'auth' => $this->auth,
             ($this->type === 'application/json' ? 'json' : 'body') => $this->body,
         ];
+        if (!empty($this->requestOptions)) {
```

Comment:
I can do that, on a closer look the statements are probably not even necessary,  as there are default defined requestOptions in Client.